docs/ecosystem/community.md

```diff
@@ -9,186 +9,17 @@ Sponsors, contributors, and adopters make this ecosystem possible and shape it i
 Take part in the Ory Community!
 
 - Chat on the [Ory Community Slack](https://slack.ory.sh/)
-- Discuss or share code on [GitHub](https://github.com/orgs/ory/discussions)
 
-Each major project also has its own discussion board:
```

Comment:
Should we keep these links here for people to engage? Otherwise this page is pretty empty and can probably removed all together

Reply:
It moved to the bottom, I see

Reply:
yea we can think about if we want to keep this page - or rather merge it with one of the other ones 🤔 have not had a good idea yet. 

right now it would mostly link to the repo containing all the links to community projects